transcendence-app/src/shared/db/postgres/utils.ts

```diff
@@ -21,21 +25,18 @@ export const makeQuery = async <T>(
 ): Promise<T[] | null> => {
   try {
     const res = await pool.query(query);
-    return res?.rows?.length ? (res.rows as T[]) : null;
-  } catch (e) {
-    // TODO temp solution until we implement logging
-    throw new Error(stringifyPostgresError(e));
+    return res.rows;
+  } catch (err) {
+    if (isCriticalDatabaseError(err)) {
```

Comment:
Here we log database errors only for critical cases:
FATAL -> Reports an error that caused the current session to abort.
PANIC -> Reports an error that caused all sessions to abort.

We don't log errors for these cases:
ERROR -> Reports an error that caused the current command to abort.

https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS